PowerShell Contributing.md

```diff
@@ -0,0 +1,305 @@
+# Ten ways of contributing to PowerShell OSS
+
+## How
+
+You can contribute by lending your time and expertise. There are so many ways to do so. Blogging, speaking, teaching, sharing code, writing modules, helping others on forums/channels to name a few. In this blog I want to focus on contributing to the PowerShell language directly. 
```

Comment:
General comment on Markdown formatting before I forget!

Markdown doesn't treat a single line break as a new paragraph and instead converts it to a space, so it's often much easier to edit, reference, and review the Markdown if you have a reasonably consistent 1-sentence-per-line rule. 

You can do this with a find/replace if you wish; VS Code's find/replace and I'm sure other editors do as well -- quick regex to find sentence endings is `\.[ ]` and you can replace that with a linebreak or `\n` if regex tokens are accepted in the replacement box in your editor.

I'll drop back later this evening or in the next couple days to do a proper review, but my initial skim looks promising. Can't wait! 😊 💖